src/Gir/Marshal/Namespace.cs

```diff
@@ -5,6 +5,6 @@ namespace Gir
 {
 	public partial class Namespace
 	{
-		public IEnumerable<ISymbol> GetSymbols() => Utils.GetAllCollectionMembers<ISymbol>(this);
+		public IEnumerable<ISymbol> GetSymbols() => Utils.GetAllCollectionMembers<ISymbol> (this);
```

Comment:
We are inconsistent on whether there is a space or now here. Do we want to have a space or no space?

Reply:
Not sure, VSMac doesn't support editorconfig properly in master, afaik